battan40/KittensToWitches
# Feature/readme

### Is this a fix or a feature?
* Feature

### What is the change or fix?
* Adds a readme
* icons
* deploy links
* git hub links
* further education links

### How should this be tested?
* open and look at readme 
* make sure all images and links are clickable

### Linked Issues:
*
#32 

DON'T FORGET THE GIPHY
